src/v17/ast.rs

```diff
@@ -50,6 +50,7 @@ pub enum Sequential {
     Event(EventTy, Expr),
     If(SequentialIfElse),
     Assert(Expr, Option<Rc<Sequential>>),
+    ExprStmt(Expr),
```

Comment:
What do you think on calling it `SeqExpr`? I know is a `stmt` but this way we know it can only be sequential and we don't get confused with module-stmts

Reply:
done